packages/components/src/InputFile/InputFile.tsx

```diff
@@ -180,7 +196,18 @@ export function InputFile({
     options.accept = allowedTypes.join(",");
   }
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone(options);
+  const { getRootProps, getInputProps, isDragActive, fileRejections } =
```

Comment:
I'll preface this with the fact that I know I'm asking to add some more (design) scope, so feel free to push back

if we have the `fileRejections` and know if something didn't work out, I wonder if we should be adding a red border or something to the overall dropzone

definitely more of a @chris-at-jobber question/solution though